src/components/chatItem/index.ts

```diff
@@ -1,9 +1,39 @@
 import Component from "../../core/Component";
 import tpl from "./tpl";
 import "./chatItem.scss";
+import { ChatsAPI } from "../../api";
+import { Actions } from "../../core/Store";
 
 export default class ChatItem extends Component {
   render() {
     return this.compile(tpl);
   }
+
+  addEvents() {
+    const activeChatBtn = this._element.querySelector("#activate-chat") as HTMLButtonElement;
+    const deleteChatBtn = this._element.querySelector("#delete-chat") as HTMLButtonElement;
+    const chatItemWrap = this._element.querySelector(".chat-item__wrap") as HTMLButtonElement;
```

Comment:
querySelector можно типизировать.
Явное приведение типа лучше избегать, потому что это небольшой "костыль", это значит, что мы что-то делаем не так

```suggestion
    const activeChatBtn = this._element.querySelector<HTMLButtonElement>("#activate-chat");
    const deleteChatBtn = this._element.querySelector<HTMLButtonElement>("#delete-chat");
    const chatItemWrap = this._element.querySelector<HTMLButtonElement>(".chat-item__wrap");
```